molecule/kafka-connect-replicator-plain-kerberos-rhel/molecule.yml

```diff
@@ -233,8 +233,6 @@ provisioner:
         kafka_connect_replicator_consumer_ssl_cert_path: "{{ lookup('env', 'MOLECULE_SCENARIO_DIRECTORY') }}/generated_ssl_files/{{inventory_hostname}}-ca1-signed.crt"
         kafka_connect_replicator_consumer_ssl_key_path: "{{ lookup('env', 'MOLECULE_SCENARIO_DIRECTORY') }}/generated_ssl_files/{{inventory_hostname}}-key.pem"
         kafka_connect_replicator_consumer_ssl_key_password: keypass
-        kafka_connect_replicator_truststore_storepass: confluenttruststorepass
```

Comment:
We expect these properties to get overwritten. Why are we removing from molecule files.

Reply:
Same property with same name was given twice in the file. That was generating the yamllint warnings.
https://github.com/confluentinc/cp-ansible/blob/353ddc442e4eda5b023a9f257a571a01a0fd76e4/molecule/kafka-connect-replicator-plain-kerberos-rhel/molecule.yml#L217-L232

Reply:
Thanks!